documentation/data-importer.adoc

```diff
@@ -117,80 +59,73 @@ OrderID,ProductID,UnitPrice,Quantity,Discount
 
+=== Setting a node ID
 
-In this example, we can shorthen the `orderID` property to simply read `id` since the fact it's on the `Order` node makes this self-evident.
-To do this, simply click the *pencil icon* against the `orderId` property, change the property name to `id` and confirm the change by pressing the *check mark*.  If you inspect the `File mapping` tab, you'll see this `id` property is still mapped to the `orderId` column.
+Nodes need an ID to define relationships with other nodes. An ID also ensures an import doesn't create a node more than once if it's referenced multiple times in a data file. 
 
-// TODO: Can we get these icons workign and inserted at appropriate place in above paragraph
-// icon:PencilIconOutline[]
-// icon:CheckIconOutline[]
+* Return to the _Definition_ tab and note that the `id` property is set as the ID for the node highlight:import/import-node-id[ID]. 
+* The import tool did this for you automatically based on the property name containing the `id` string. In other cases, you may need to correct it or set it manually.
 
-=== Select an ID
+[TIP]
+====
+You can also specify properites and their data types in isolation from mapping by using the highlight:import/import-property-add-button[add property] button. You can then map a column from your data file to the property in the file mapping tab.
+====
 
-Returning to the _Definition_ tab, the last thing you need to do is to select one of the properties as the node highlight:import/import-node-id[ID]. The prupose of selecting an ID for a node is two fold:
 
-. If the node is referenced more than once in the file, the ID ensures that the node is not created more than once.
-. The node ID is used when connecting nodes with relationships. You'll learn more about this soon, as either a start node or end node.
+=== Create the Product node 
 
-It should be fairly self-evident here that we need to select the `id` property as the ID for the node. 
-In this example, Data Importer has been helpful and selected it for you automatically (it guessed based on the property name containing the `id` string), but in other cases you may need to select the `id`` manually or correct an id Data Importer guesses for you.
+Now you just need to repeat the process you followed for `Order`, creating a `Product` node from the `products.csv` file. 
+Here you need to map the `produtId` and `productName` columns, rename the `productId` property to be simply `id` and set it to be a `string`. Ensure `id` is selected as the ID.
 
-Now you just need to repeat the process, creating a `Product` node from the `products.csv` file. Here you need to map the `produtId` and `productName` columns, rename the `productId` property to be simply `id` in a similar way we did for the `Order` node and ensure `id` selected as the ID.
+[TIP]
+====
+You can rapidly add nodes by hovering over an existing node; when a green plus icon appears, click-drag from the node and then release onto the canvas. 
```

Comment:
Maybe more specifically this could say "You can rapidly add nodes with a linking relationship" since it also creates the relationship and not just a node.

The very similar behaviour for linking two nodes is covered soon after in the next section, maybe it can be moved as a tip for that, loosely speaking "a way to create a new node with a relationship rather than link two existing".